tests/system/provisioning/roles/aws/tests/route53_records.yml

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
This file can be deleted as it was imported from demo-environment but it's not used in this deployment.

Reply:
Done e02cb79daa213d847593a9afe11e23edf5481672